spec/features/book_index_spec.rb

```diff
@@ -50,12 +53,12 @@
   end
 
   it "user_can_see_all_authors" do
-    author_1 = @book_1.reviews.create(title: "Good book", score: 4, review_text: "text body")
-
+    book_4 = @author_1.books.create(title: "Hamlet", pages: 342, year_pub: 1992, image: "https://images-na.ssl-images-amazon.com/images/I/41IETeONh-L._SX331_BO1,204,203,200_.jpg", publisher: "Simon & Schuster")
```

Comment:
Then I don't think we'd create this way. Whereas a book can only have 1 review (so a review has a book_id), authors can have many books so have no book_id so I don't think you can create this way. Have to use the book_authors connection.